pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -1282,6 +1289,50 @@ func triggerPublishDevInfo(ctxPtr *zedagentContext) {
 	}
 }
 
+func triggerPublishAllInfo(ctxPtr *zedagentContext) {
+
+	log.Function("Triggered PublishAllInfo")
+	go func() {
+		ctxPtr.TriggerInfo <- infoForObjectKey{infoType: info.ZInfoTypes_ZiDevice}
```

Comment:
In fhat separate channel and function this would just become a call to triggerPublishDevInfo().